index.html

```diff
@@ -25,7 +26,121 @@ <h1 class="header2">I'm Ali<br>Glad to see<br>you!</h1>
+                        <ul class="programs">
+                            <li class="html"><a href="https://"><img src="img/html.png" alt=""></a></li>
+                            <li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>
+                            <li class="javascript"><a href="https://"><img src="img/js.png" alt=""></a></li>
+                        </ul>
+                        <a class="see_project" href="https://"><img  src="img/see_project.png" alt="see projects"></a>
+                    </section>
+        <section class="main_body">
+            <h1 class="header2">About<br>Myself</h1>
+            <p class="body2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">Let's Connect</p>
+            <ul class="icons">
+                <li class="twitter"><a href="https://"><img src="img/Icon - Twitter.png" alt="Twitter"></a></li>
+                <li class="linkedin"><a href="https://"><img src="img/Icon - Linkedin.png" alt="Linkedin"></a></li>
+                <li class="medium"><a href="https://"><img src="img/Icon - Medium.png" alt="Medium"></a></li>
+                <li class="github"><a href="https://"><img src="img/Icon -GitHub.png" alt="GitHub"></a></li>
+                <li class="angelist"><a href="https://"><img src="img/Icon - Angelist.png" alt="Angelist"></a></li>
+            </ul>
+            <a class="see_project" href="https://"><img  src="img/get_resume.png" alt="see projects"></a>
+        </section>
+        <section class="languages">
+            <ul class="lang_info_head">
+                <li class="lang_info"><p >languages</p> </li>
+                <li class="vector"><i class="fa-solid fa-angle-down"></i></li>
+            </ul>
+            <ul class="prog_lang">
+                <li class="cycle_js"><a href="http://"><img src="img/Frame 9.png" alt=""></a></li>
+                <li class="cycle_html"><a href="http://"><img src="img/Frame 10.png" alt=""></a></li>
+                <li class="cycle_css"><a href="http://"><img src="img/Frame 11.png" alt=""></a></li>
+            </ul>  
+            <ul class="lang_info_head">
+                <li class="lang_info"><p >Frameworks</p> </li>
+                <li class="vector_frameworks"><i class="fa-solid fa-angle-right"></i></li>
+            </ul>
+            <p class="sep"><i class="fa-thin fa-horizontal-rule"></i></p>
+            <ul class="lang_info_head">
+                <li class="lang_info"><p >Skills</p> </li>
+                <li class="vector_skills"><i class="fa-solid fa-angle-right"></i></li>
+            </ul>
+            <p class="sep"><i class="fa-thin fa-horizontal-rule"></i></p>
+        </section>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`. This is a project requirement. Do not put each of them in a separate `ul` tag as it is now. 🤝